packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/test/feature_counter/blocs/counter_bloc_test.mocks.dart

```diff
@@ -1,30 +1,49 @@
-// Mocks generated by Mockito 5.0.1 from annotations
-// in example/test/bloc_tests/counter_bloc_test.dart.
+// Mocks generated by Mockito 5.0.9 from annotations
+// in test_app/test/feature_counter/blocs/counter_bloc_test.dart.
 // Do not manually edit this file.
 
-import 'dart:async' as _i3;
+import 'dart:async' as _i5;
 
-import 'package:{{project_name}}/base/repositories/counter_repository.dart'
-    as _i2;
 import 'package:mockito/mockito.dart' as _i1;
+import 'package:{{project_name}}/base/count_data_sources/count_data_source.dart' as _i2;
```

Comment:
When I generate the project an error in shown in this line. Perhaps instead of `import 'package:test_app/base/count_data_sources/count_data_source.dart' as _i2;` we should have something like `import 'package:test_app/base/data_sources/domain_data_sources/count_data_sources/count_data_source.dart' as _i2;` ?

Reply:
Edited.